data-app/db/Models.py

```diff
@@ -132,7 +132,28 @@ def __repr__(self) -> str:
         return retVal
 
 
-engine = create_engine("sqlite:///../ProcessedData.db",echo=False,future=True)
+engine = create_engine(
+    "sqlite:///../ProcessedData.db",
+    echo=False,
+    future=True,
+    pool_pre_ping=True,
+    pool_recycle=300,
+    connect_args={
+        'timeout': 30,
+        'check_same_thread': False
+    }
+)
```

Comment:
Default settings are usually fine here, I don't think we need these additional args? What led you to make these decisions?